api/src/main/java/uk/gov/hmcts/payment/api/configuration/RestTemplateConfiguration.java

```diff
@@ -20,4 +20,12 @@ public CloseableHttpClient paymentsHttpClient() {
     public RestTemplate restTemplatePaymentGroup() {
         return new RestTemplate(new HttpComponentsClientHttpRequestFactory());
     }
+
+    @Bean (value = "restTemplateIacSupplementaryInfo")
+    public RestTemplate restTemplateIacSupplementaryInfo() {
+        var factory = new HttpComponentsClientHttpRequestFactory();
+        factory.setConnectTimeout(5000);
```

Comment:
Can we configure in properties file for value?

Reply:
resolved